MonoGame.Framework/Vector3.cs

```diff
@@ -392,9 +392,9 @@ public static void Distance(ref Vector3 value1, ref Vector3 value2, out float re
         /// <returns>The squared distance between two vectors.</returns>
         public static float DistanceSquared(Vector3 value1, Vector3 value2)
```

Comment:
We need a unit test for this as there isn't one and we need to be sure this didn't break anything.